dev-util/clion/clion-2017.3.4.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop
+inherit eutils
+
+SLOT="0"
```

Comment:
Please keep variables in standard order:

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE

RDEPEND
```

Reply:
done